packages/app/src/components/editor/ArticleWizardStep2.tsx

```
@@ -0,0 +1,75 @@
+      {coverImage ? (
+        <div className="border border-neutral-900 my-4 overflow-hidden">
+          <img
+            alt={title}
+            className="grayscale object-cover w-full"
+            src={coverImage}
+          />
+        </div>
+      ) : null}
+
+      <div className={"mt-2 prose max-w-none text-[16px] text-neutral-900 leading-relaxed bg-panel"}>
+        <div className="whitespace-pre-wrap" dangerouslySetInnerHTML={{ __html: renderMarkdownToHtml(content) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)